src/wp-includes/class-wp-scripts.php

```diff
@@ -319,7 +319,14 @@ public function do_item( $handle, $group = false ) {
 			$after_non_standalone_handle = $this->print_inline_script( $handle, 'after-non-standalone', false );
 
 			if ( $after_non_standalone_handle ) {
-				$after_handle .= sprintf( "<script%s id='%s-js-after' type='text/template'>\n%s\n</script>\n", $this->type_attr, esc_attr( $handle ), $after_non_standalone_handle );
+				$after_handle               .= sprintf(
+					'<script%1$s id=\'%2$s-js-after\' type=\'text/template\' data-wp-executes-after=\'%2$s\'>%4$s%3$s%4$s</script>%4$s',
+					$this->type_attr,
+					esc_attr( $handle ),
+					$after_non_standalone_handle,
+					PHP_EOL
```

Comment:
```suggestion
					'<script%s id=\'%2$s-js-after\' type=\'text/template\' data-wp-executes-after=\'%s\'>\n%s\n</script>\n',
					$this->type_attr,
					esc_attr( $handle ),
					$after_non_standalone_handle,
```

Similar to other scripts, use `\n` instead of `PHP_EOL` and use `%s` as it is sufficient for these changes as we don't need to add translators or change the order of the arguments.

Reply:
@mukeshpanchal27  I was trying the same initially but \n don't work when the parent string is in single quotes. It is instead printed as string `\n`.  Numbered placeholder was added as there are multiple places i wanted the handle string and for numbered placeholder I need the enclosing string to be in single quotes.